lib/samples/create_buffers_around_points/create_buffers_around_points.dart

```diff
@@ -0,0 +1,446 @@
+        20.0,
+        max(
+          20.0,
+          View.of(context).viewPadding.bottom /
+              View.of(context).devicePixelRatio,
+        ),
+      ),
+      child: Column(
+        mainAxisSize: MainAxisSize.min,
+        children: [
+          Row(
+            children: [
+              Text(
+                'Settings',
+                style: Theme.of(context).textTheme.titleLarge,
+              ),
+              const Spacer(),
+              IconButton(
+                onPressed: () {
+                  Navigator.pop(context); // Close the bottom sheet
+                },
+                icon: const Icon(Icons.close),
+              ),
+            ],
+          ),
+          Row(
+            children: [
+              const Text('Buffer Radius (miles)'),
+              const Spacer(),
+              Text(
+                _bufferRadius.round().toString(),
+                textAlign: TextAlign.right,
+              ),
+            ],
+          ),
+          Row(
+            children: [
+              Expanded(
+                // A slider to adjust the buffer radius.
+                child: Slider(
+                  value: _bufferRadius,
+                  min: 10,
+                  max: 300,
+                  onChanged: (value) => setState(() {
+                    _bufferRadius = value;
+                  }),
+                ),
+              ),
+            ],
+          ),
+          Row(
+            children: [
+              Text(_shouldUnion ? 'Union Enabled' : 'Union Disabled'),
+              const Spacer(),
+              Switch(
+                value: _shouldUnion,
+                onChanged: (value) {
+                  setState(() {
+                    _shouldUnion = value;
+                    if (_bufferPoints.isNotEmpty) {
+                      drawBuffers(unionized: _shouldUnion);
+                    }
+                  });
+                },
+              ),
+            ],
+          ),
+        ],
+      ),
+    );
+  }
+
+  void onMapViewReady() async {
+    // Initialize the boundary polygon and the symbols.
+    _boundaryPolygon = _makeBoundaryPolygon();
+    _initializeSymbols();
+
+    // State Plane Central Texas Spatial reference.
+    final statePlanNorthCentralTexasSpatialReference =
+        SpatialReference(wkid: 32038);
+
+    // Create a map with the spatial reference as the basemap and add it to our map controller.
+    final map =
+        ArcGISMap(spatialReference: statePlanNorthCentralTexasSpatialReference);
+
+    // Add some base layers (counties, cities, highways).
+    final mapServiceUri = Uri.parse(
+      'https://sampleserver6.arcgisonline.com/arcgis/rest/services/USA/MapServer',
+    );
+
+    // Create a map image layer from the uri.
+    final usaLayer = ArcGISMapImageLayer.withUri(mapServiceUri);
+
+    // Add the map image layer to the map.
+    map.operationalLayers.add(usaLayer);
+
+    // Load the map image layer.
+    await usaLayer.load();
```

Comment:
Is it necessary to manually load the layer here? I think it'll get loaded automatically just fine.

Reply:
If we can do without this line, then onMapViewReady doesn't have to be declared async.